packages/react-dom/src/__tests__/ReactDOMInput-test.js

```diff
@@ -1433,6 +1433,40 @@ describe('ReactDOMInput', () => {
       expect(node.getAttribute('value')).toBe('2');
     });
 
+    it('does not set the value attribute on password inputs', () => {
+      const Input = getTestInput();
+      const stub = ReactTestUtils.renderIntoDocument(
+        <Input type="password" value="1" />,
```

Comment:
Side note. `Input` here, as a React component, through me off for a bit. I wonder if we should rename this `TestInput` or `ControlledInput`.